.env

```diff
@@ -1,5 +1,8 @@
 # ODM product version
-ODMVERSION=8.9.1
+ODMVERSION=vnext
```

Comment:
Why this change ?. The version should be the version that a customer use with their ODM Version. As discuss this morning we should probably rename it to  something like : odmversion